walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

```diff
@@ -44,7 +45,8 @@ SSO uses *Realms* to manage *Clients*, *Roles*, *Users* and *Groups*. A
 user belongs to and logs into a realm. Realms are isolated from one another and
 can only manage and authenticate the users that they control.
 
-A realm has already been created for this Solution Pattern and user.
+NOTE: The `walkthroughs` realm used in this Solution Pattern is shared with all users on the cluster. Do not use this realm for sensitive applications.
```

Comment:
I'd change this to say "production applications" or "Only use this realm for Solution Pattern content.  Do not use this realm for production applications."

Reply:
went with 'production applications' option